### PR TITLE
Add documentation about hvac limitation on ZE50

### PR DESCRIPTION
**Pull request by PysX**
Following #51 : having HVAC status does not work for the ZE50 model causes returns of issues.
Climate entity is now disabled for this model and some documentation could help user to use `renault.ac_start` service as this one works.

### CODE REVIEW

File: CONFIGURE.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Limitations
```
Comment (epenet) [inline CONFIGURE.md:1]:
```suggestion
# Configuration
```

File: CONFIGURE.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+The component tends to be generic and adapt to all supported models. However, Renault locks or limit some functionnalities based on vehicle model.
+This page list them.
```
Comment (epenet) [inline CONFIGURE.md:4]:
```suggestion
The component is generic and tries to adapt to all supported models. It is mostly configured via the UI however, due to limitations on the Renault API, some features may need to be configured manually.
```

File: CONFIGURE.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## HVAC
+Hvac status is not available for Zoe50 (model code X102VE). But Starting HVAC is possible with service `renault.ac_start`.
```
Comment (epenet) [inline CONFIGURE.md:7]:
```suggestion
On some models (eg. Zoe40), HVAC status always reports as off. On other models (eg. Zoe50) it is not available at all. HVAC control is therefore not available as a standard climate entity, but needs to be controlled via a service call to `renault.ac_start`.
```

File: CONFIGURE.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+The service `renault.ac_stop` does not provide errors, but it is assumed that is as no effect on the vehicle.
```
Comment (epenet) [inline CONFIGURE.md:9]:
```suggestion
Please note that the service `renault.ac_stop` does not provide errors, but appears to have no effect on some models.
```

File: CONFIGURE.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+Following MyRenaultApp documentation, HVAC is started between 5 and 50 min.
+
+This is an example script to start HVAC for Zoe50 :
```
Comment (epenet) [inline CONFIGURE.md:12]:
```suggestion
This is an example script to start HVAC :
```

File: CONFIGURE.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+    message: Starting HVAC at 21°C
+    title: Renault ZOE
+  # optional delay to avoid multiple launch 
+  - delay:
+      seconds: 300
+  mode: single
+  icon: mdi:snowflake
+```
+
+This is and example of switch :
```
Comment (epenet) [inline CONFIGURE.md:36]:
```suggestion
This is an example of a fake switch, whose state will match whether the script is still running (ie. was started less than 300 seconds ago) :
```

File: CONFIGURE.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+```yaml
+# Switches
+switch:
+  - platform: template
+    switches:
+      # Start AC now for five minutes (see script)
+      zoe_ac_start:
+        friendly_name: "A/C"
+        icon_template: mdi:fan
+        value_template: "{{ is_state('script.zoe_ac_start', 'on') }}"
```
Comment (epenet) [inline CONFIGURE.md:46]: Should this not match the script name `script.start_hvac` like turn_on/turn_off ?

**Review by epenet — COMMENTED** [review]
Looks good, can you please update the `Configuration` section in the README to add a link to this new page?

**Comment by PysX** [comment]
Thank you very much for review and a better much english !

File: README.md [hunk 8]
```diff
@@ -53,9 +53,9 @@ custom_components/renault/services.yaml
 custom_components/renault/strings.json
 ```
 
-## Configuration is done in the UI
+## Configuration
+Configuration is done in the UI, but some cases needs [specific configuration](CONFIGURE.md).
```
Comment (epenet) [inline README.md:57]: One last minor tweak:

```suggestion
Configuration is done in the UI, but some cases need [specific configuration](CONFIGURE.md).
```

**Comment by epenet** [comment]
Thanks @PysX

**Pull request merged**